spacetemplate/assets/legacy.yaml

```diff
@@ -31,7 +34,9 @@ work_item_types:
   extends: "86af5178-9b41-469b-9096-57e5155c3f31"
   name: Feature
   can_construct: yes
-  description: TBD
+  description: Features, often expressed as user stories, are required to
```

Comment:
For multi-line strings please use `>` and indentation as done in the space template's description.

Reply:
Fixed